files/__addonLocation__/package.json

```diff
@@ -25,6 +25,9 @@
     "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
     "prepack": "rollup --config"
   },
+  "peerDependencies": {
```

Comment:
@simonihmig now only ember-source is a peer.

you are correct that it doesn't make sense for us to include `@glimmer/component` as a peer by default (as many utility libraries may not care about `@glimmer/component` at all!)